### PR TITLE
add recording rules

### PR DESCRIPTION
**Pull request by jessegoodier**
@bstuder99 Please verify this anchor is done per gitbook standards?

@kubecost/solutions-engineers please validate config

### CODE REVIEW

**Comment by bstuder99** [comment]
What exactly are you trying to do with the anchor? All H2 and higher headers should appear naturally as anchors along the right navigation of an article, such as when formatted as `## Example title`

**Comment by jessegoodier** [comment]
> What exactly are you trying to do with the anchor? All H2 and higher headers should appear naturally as anchors along the right navigation of an article, such as when formatted as `## Example title`

there will be a link to #recording-rules in the 1.102 release notes

**Comment by bstuder99** [comment]
Ok, I see. Not super good with this, but if we're trying to merge this before the docs go live, you can add this link to the release notes and it should work: https://docs.kubecost.com/install-and-configure/install/custom-prom#recording-rules

**Comment by jessegoodier** [comment]
@michaelmdresser Does this look right to you?

File: custom-prom.md [hunk 1]
```diff
@@ -67,6 +67,29 @@ Kubecost requires the following minimum versions:
 
 This config needs to be added to `extraScrapeConfigs` in the Prometheus configuration. Example [extraScrapeConfigs.yaml](./images/extraScrapeConfigs.yaml)
 
+<a name="recording-rules"></a>
```
Comment (srpomeroy) [inline custom-prom.md:70]:
```suggestion
## Recording rules
```

Comment (srpomeroy) [reply custom-prom.md:70]: Proper MD syntax

Comment (jessegoodier) [reply custom-prom.md:70]: but it isn't a new section. it is part of #2

Comment (bstuder99) [reply custom-prom.md:70]: Just correct it to ###. The link I posted above will still work to direct users to this anchor

Comment (jessegoodier) [reply custom-prom.md:70]: like this?

**Review by bstuder99 — APPROVED** [review]

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**